Append sq payment form script to document

spenweb opened the pull request:
Resolves #81 

The Square payment form library script was not being appended to the document which prevented the form to load. This PR adds back what was previously there to add the script to the document.

HarounAns commented:
I don't believe this fixes the issue. I ran your branch locally and still got the issue
![image](https://user-images.githubusercontent.com/20847319/92339208-f9779580-f082-11ea-827f-5567049debfc.png)
![image](https://user-images.githubusercontent.com/20847319/92339217-01cfd080-f083-11ea-8c0f-c53be583a567.png)

spenweb commented:
@stephtaylor are there any special instructions on how to get a locally built package for development purposes running inside of a project? These are the steps I performed in my attempt:
1. clone this repo
1. `npm run build` inside of this repo
1. Removed package from my project that requires this package (`yarn remove react-square-payment-form`)
1. Added the built local package to my project using a file path (`yarn add react-square-payment-form@file:/path/to/local/repo/react-square-payment-form`
1. Tested the payment form page (this is where I got many errors in my create-react-app)

spenweb commented:
@stephtaylor I think I figured out how to get a locally built package into my project. I remove the folders that are mentioned in the [.npmignore](https://github.com/square/react-square-payment-form/blob/master/.npmignore) file. After that, my project was able to load and my change that is in this PR resolved my issue of the payment form elements not loading.

@HarounAns, I was able to successfully get the change to load and it resolved the issue for my project.
